sass/_watch.scss

```diff
@@ -137,64 +137,72 @@
     .talks-grid {
         display: grid;
         grid-template-columns: repeat(auto-fill, 358px);
+        grid-template-rows: repeat(auto-fit, 533px);
```

Comment:
Where does this 533 come from?

If I reduce it to say 490 (probably not the optimal number), it seems like my first comment is resolved
![image](https://github.com/user-attachments/assets/2f7ae696-1feb-48f8-a538-3bdb6ebbb0b2)

Reply:
It was the number Firefox calculated for the card height of the  old/deployed grid.

Reply:
Then I think we could probably diverge from it? Can you repro what I mean?